internal/commands/scan.go

```diff
@@ -758,6 +767,14 @@ func setupScanTypeProjectAndConfig(
 	if apiSecConfig != nil {
 		configArr = append(configArr, apiSecConfig)
 	}
+
+	var SCSConfig, scsErr = addSCSScan(cmd)
+	if scsErr != nil {
+		return scsErr
```

Comment:
why do you return here??

Reply:
Because we need additional flags when we want to perform a Scorecard scan (scs-repo-token and scs-repo-url), we return an error if any of those are missing. Don't worry, this only happens if the user specified scs in as a scan type.